ExpensesWebApplication/App.aspx

```diff
@@ -56,8 +56,8 @@
         const DAYS_IN_A_WEEK_FLOAT = 7.0;
         const MONTHS_IN_A_YEAR_DECIMAL = 12.0;
         const MONTHS_IN_A_HALFYEAR_DECIMAL = 6.0;
-        const AVERAGE_WEEKS_PER_MONTH_DECIMAL = 30.437 / 7.0;
-        const AVERAGE_BIWEEKS_PER_MONTH_DECIMAL = 30.437 / 14.0;
+        const AVERAGE_WEEKS_PER_MONTH_DECIMAL = 4.345;
+        const AVERAGE_BIWEEKS_PER_MONTH_DECIMAL = 4.345 / 2;
```

Comment:
There is no performance loss since this is simplified when the program is compiled.  In terms of "should you," I believe is subjective.  I leave it displayed so it is more clear to the developer reviewing it what the formula is to retrieve these magic numbers.